src/overlay/OverlayManagerImpl.cpp

```diff
@@ -1115,4 +1149,191 @@ OverlayManagerImpl::updateFloodRecord(StellarMessage const& oldMsg,
     ZoneScoped;
     mFloodGate.updateRecord(oldMsg, newMsg);
 }
+
+size_t
+OverlayManagerImpl::getMaxDemandSize() const
+{
+    auto const& cfg = mApp.getConfig();
+    double ledgerCloseTime =
+        std::chrono::duration_cast<std::chrono::milliseconds>(
+            cfg.getExpectedLedgerCloseTime())
+            .count();
+    double demandTime = cfg.FLOOD_DEMAND_PERIOD_MS.count();
+    double queueSize =
+        mApp.getHerder().getMaxQueueSizeOps() * cfg.FLOOD_OP_RATE_PER_LEDGER;
+
+    size_t res = std::ceil(queueSize * demandTime / ledgerCloseTime);
```

Comment:
please do not do division with doubles and conversion to size_t like this, as it causes overflow and undefined behavior. Use bigDivideOrThrow instead, see TransactionQueue::getMaxOpsToFloodThisPeriod() as an example.

Reply:
I used `getMaxOpsToFloodThisPeriod` as an example and now `Wconversion` doesn't show any warnings.